package.json

```diff
@@ -13,7 +14,9 @@
     "test": "ng test @politie/ngx-sherlock",
     "test:ci": "ng test @politie/ngx-sherlock --no-watch --browsers=ChromeHeadless --code-coverage",
     "lint": "ng lint",
-    "e2e": "ng e2e"
+    "e2e": "export NODE_OPTIONS=--openssl-legacy-provider && ng e2e",
```

Comment:
After the removal of a call to this script from the workflow, I don't think we need this anymore. Can it be removed?